recipes/core/busybox.yaml

```diff
@@ -31,6 +31,20 @@ buildScript: |
     makeParallel
     make CONFIG_PREFIX=${BOB_CWD}/install install
 
-packageScript: |
-    cp -a $1/install/* .
+    mkdir -p install/.bob
+
+    # busybox must be setuid
+    cat >install/.bob/busybox.device-table <<EOF
```

Comment:
Theoretically we should test for `CONFIG_FEATURE_SUID`. Otherwise that would create a security hole. OTOH, I don't think anybody will ever disable this feature.

Reply:
done